src/application.rs

```diff
@@ -32,7 +32,7 @@ pub struct CoLink {
     pub(crate) ca_certificate: Option<Certificate>,
     pub(crate) identity: Option<Identity>,
     #[cfg(feature = "variable_transfer")]
-    pub(crate) vt_p2p: Arc<crate::extensions::variable_transfer::p2p_inbox::VTP2P>,
+    pub(crate) vt_p2p: Arc<crate::extensions::variable_transfer::p2p_inbox::VTP2PCTX>,
```

Comment:
Just a thought: should we consider adding underscores? e.g. `VT_P2P_CTX`
Just realized VTP2PCTX looks... hard to parse. What do you think?

Reply:
https://doc.rust-lang.org/1.0.0/style/style/naming/README.html

Reply:
Yeah, I'm aware of this... so I'm saying VTP2PCTX is hard to parse as a special case.
If you really tend to follow it, maybe we should use `VarTransP2pCtx` or `VtP2pCtx` to follow the CamelCase guideline exactly.